src/main/locales/gen_defines.py

```
@@ -0,0 +1,144 @@
+    wr_bf_header(fw_to, bf_header_name)
+    wr_bf_notice_bf(fw_to, locale)
+
+if task == 'UT':
+    fw_ut = open(untrans_filename, 'w')
+    wr_bf_header(fw_ut, bf_header_name)
+    wr_bf_notice_ut(fw_ut)
+
+prevKey = ""
+prevMessage = prevDescription = ""
+
+tree = ET.parse(from_filename)
```

Ruff (0.12.2)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)